### PR TITLE
Update rubygem-smart_proxy_container_gateway to 1.0.7

### PR DESCRIPTION
**Pull request by ianballou**
Looks like the `./bump_rpm.sh` made some major changes to the spec to move away from SCL, hopefully all is well there.

### CODE REVIEW

**Comment by evgeni** [comment]
> Looks like the `./bump_rpm.sh` made some major changes to the spec to move away from SCL, hopefully all is well there.

Nope, it used the wrong template :(

File: packages/plugins/rubygem-smart_proxy_container_gateway/rubygem-smart_proxy_container_gateway.spec [hunk 1]
```diff
@@ -1,76 +1,54 @@
...
-Name: %{?scl_prefix}rubygem-%{gem_name}
-Version: 1.0.6
+Name: rubygem-%{gem_name}
+Version: 1.0.7
 Release: 1%{?foremandist}%{?dist}
 Summary: Pulp 3 container registry support for Foreman/Katello Smart-Proxy
-Group: Development/Languages
 License: GPLv3
-URL: http://github.com/ianballou/smart_proxy_container_gateway
+URL: https://github.com/ianballou/smart_proxy_container_gateway
```
Comment (evgeni) [inline packages/plugins/rubygem-smart_proxy_container_gateway/rubygem-smart_proxy_container_gateway.spec:16]: https://github.com/Katello/smart_proxy_container_gateway/pull/22 ;-)

**Comment by evgeni** [comment]
Should be good to go now, but I kinda want someone else to review ;)

File: packages/plugins/rubygem-smart_proxy_container_gateway/rubygem-smart_proxy_container_gateway.spec [hunk 2]
```diff
@@ -1,76 +1,54 @@
...
-
+# template: smart_proxy_plugin
 %global gem_name smart_proxy_container_gateway
 %global plugin_name container_gateway
-%global gem_require_name %{gem_name}
-%global foreman_proxy_dir %{_root_datadir}/foreman-proxy
+
+%global foreman_proxy_min_version 1.24
+%global foreman_proxy_dir %{_datadir}/foreman-proxy
+%global foreman_proxy_statedir %{_localstatedir}/lib/foreman-proxy
```
Comment (ekohl) [inline packages/plugins/rubygem-smart_proxy_container_gateway/rubygem-smart_proxy_container_gateway.spec:7]: I noticed we can do better here: https://github.com/theforeman/foreman-packaging/pull/8807

**Review by ekohl — APPROVED** [review]
Do we want to fix the statedir definition or merge it now and then update it in all packages once the template fix is merged? I'm leaning to the former.

**Comment by evgeni** [comment]
updated, thanks!

**Comment by evgeni** [comment]
I could have squashed the commits :see_no_evil:

**Pull request merged**